frontend/src/slices/auth/auth.slice.ts

```diff
@@ -30,7 +35,17 @@ const { reducer, actions, name } = createSlice({
     builder.addCase(signIn.pending, (state) => {
       state.dataStatus = DataStatus.PENDING;
     });
-    builder.addCase(signIn.fulfilled, (state) => {
+    builder.addCase(signIn.fulfilled, (state, action) => {
+      state.user = {
+        id: action.payload.id,
+        email: action.payload.email,
+        firstName: action.payload.firstName,
+        lastName: action.payload.lastName,
+        phone: action.payload.phone,
+        group: action.payload.group,
+        business: action.payload.business,
+      };
+      state.token = action.payload.accessToken;
```

Comment:
Remove it, we don't need token in store.
Instead - use thunk (sign-in action) to set this token to local storage. Use our storage service